Dockerfile

```diff
@@ -97,10 +91,10 @@ RUN wget https://github.com/Yelp/dumb-init/releases/download/v1.2.0/dumb-init_1.
 RUN dpkg -i dumb-init_*.deb && rm dumb-init_*.deb
 
 # Install cget
-RUN pip install cget
+#RUN pip install cget
```

Comment:
Do we need this command under #?

Reply:
The commands necessary for dependency install before the image was rebased to the miopen ci image have been left in, but disabled by comment.